### PR TITLE
add support for identifying node and node version

### PR DESCRIPTION
**Pull request by soliton4**
hey there, i added support for identifying node.js and its major version number.

### CODE REVIEW

File: is.js [hunk 1]
```diff
@@ -586,6 +586,19 @@
     var root = freeGlobal || freeSelf || thisGlobal || Function('return this')();
 
     var previousIs = root.is;
+  
+    // is current runtime node.js?
+    // parameter is optional
+    is.node = function(range) {
+        if (Object.prototype.toString.call(typeof process !== 'undefined' ? process : 0) === '[object process]'){
```
Comment (jdalton) [inline is.js]: There's already a variable in the source:
```js
var toString = Object.prototype.toString;
```

**Comment by soliton4** [comment]
@jdalton there i made use of it

File: is.js [hunk 2]
```diff
@@ -586,6 +586,19 @@
     var root = freeGlobal || freeSelf || thisGlobal || Function('return this')();
 
     var previousIs = root.is;
+  
+    // is current runtime node.js?
+    // parameter is optional
+    is.node = function(range) {
+        if (toString.call(typeof process !== 'undefined' ? process : 0) === '[object process]'){
```
Comment (jdalton) [inline is.js]: You can make this expression just a boolean:
```js
typeof process !== 'undefined' && process
```

Comment (soliton4) [inline is.js]: feel free to change it to whatever style you think is best

i copied the code from https://github.com/parshap/check-node-version

im not gonna make tiny changes every 10 min
you are welcome

**Comment by jdalton** [comment]
Thanks @soliton4! A PR is a bit of give and take.

**Pull request closed without merging**